From d390e79a166edd758828b110920c45b427f0e2cc Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@freescale.com>
Date: Fri, 6 Nov 2015 18:41:59 +0800
Subject: [PATCH 0665/5242] MLK-11828-1 ARM: dts: reserve OCRAM space for M4

commit  a7e9a6f272ea349e32ae243387d976ecb371c886 from
https://source.codeaurora.org/external/imx/linux-imx.git

FreeRTOS has use case of running on OCRAM, reserve the
bottom half of OCRAM spase(0x910000 - 0x91ffff) for M4.

Signed-off-by: Anson Huang <Anson.Huang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts |    4 ++++
 arch/arm/boot/dts/imx7d-sdb-m4.dts               |    4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts
index b20593c..7594c33 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-m4.dts
@@ -47,6 +47,10 @@
 	status = "disabled";
 };
 
+&ocram {
+	reg = <0x00901000 0xf000>;
+};
+
 &rpmsg{
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/imx7d-sdb-m4.dts b/arch/arm/boot/dts/imx7d-sdb-m4.dts
index f22b48b..abbe365 100644
--- a/arch/arm/boot/dts/imx7d-sdb-m4.dts
+++ b/arch/arm/boot/dts/imx7d-sdb-m4.dts
@@ -54,6 +54,10 @@
 	status = "disabled";
 };
 
+&ocram {
+	reg = <0x00901000 0xf000>;
+};
+
 &rpmsg{
 	status = "okay";
 };
-- 
1.7.9.5

